operator/pkg/helmreconciler/prune.go

```diff
@@ -139,9 +139,11 @@ func (h *HelmReconciler) DeleteObjectsList(objectsList []*unstructured.Unstructu
 				continue
 			}
 
-			err := h.client.Delete(context.TODO(), &o, client.PropagationPolicy(metav1.DeletePropagationBackground))
+			err := h.client.Delete(context.TODO(), &o,
+				client.PropagationPolicy(metav1.DeletePropagationBackground), client.GracePeriodSeconds(0))
 			if err != nil {
-				if !strings.Contains(err.Error(), "not found") {
+				if !strings.Contains(err.Error(), "not found") &&
```

Comment:
should probably use kerrors.IsNotFound and whatever the equivalent is from `could not find the requested resource`

Reply:
done